betterlint.gemspec

```diff
@@ -11,6 +11,13 @@ Gem::Specification.new do |s|
   s.summary = "Betterment rubocop configuration"
   s.description = "Betterment rubocop configuration"
   s.license = "MIT"
+
+  s.homepage = "https://github.com/Betterment/#{s.name}"
+  s.metadata["homepage_uri"] = s.homepage
+  s.metadata["source_code_uri"] = "#{s.homepage}/tree/v#{s.version}"
```

Comment:
Dynamically links directly to the tag of the release for the version in the `version.rb`, which for any release is the source of that specific release!

Set once and forget it!